src/main/java/movie/storage/Main.java

```diff
@@ -35,7 +37,8 @@ public static void main(String[] args) {
         CinemaHall secondCinemaHall = new CinemaHall();
         secondCinemaHall.setDescription("hall number 2");
         cinemaHallService.add(secondCinemaHall);
-        cinemaHallService.getAll().forEach(System.out::println);
+        cinemaHallService.getAll().forEach(cinemaHal ->
+                logger.info(cinemaHal + "is added"));
```

Comment:
you are getting cinema halls, not adding, you are throwing message about adding in DAO layer anyways. It will be enough just to use a method reference logger::info

Reply:
Agree